src/libraries/Common/src/Interop/Windows/Kernel32/Interop.DuplicateHandle_SafeProcessHandle.cs

```diff
@@ -11,9 +11,9 @@ internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, SetLastError = true, BestFitMapping = false)]
         internal static extern bool DuplicateHandle(
```

Comment:
It would be nice to rename the .cs file now that it has nothing to do with SafeProcessHandle.